specs-go/config.go

```diff
@@ -252,7 +262,7 @@ type BlockIO struct {
 }
 
 // Memory for Linux cgroup 'memory' resource management
-type Memory struct {
+type LinuxMemory struct {
```

Comment:
Doing this sort of rename on a per-collision basis (e.g. “we now have a Windows memory structure, so rename `Memory` → `LinuxMemory`”) seems like it will cause stability headaches for downstream consumers of this package.  I suggest [aggressively namespacing now][1] (and when new types are added), because “yeah, Solaris can also use `LinuxInterfacePriority`” is much easier on downstream users than “We're renaming `InterfacePriority` to `LinuxInterfacePriority` in runtime-spec v1.2”.

[1]: https://github.com/opencontainers/runtime-spec/pull/405/files/cc226d717a2eccf19dc81aee1c8a298a1ffe211c#r63821329